Document/0x06b-Basic-Security-Testing.md

```diff
@@ -96,54 +96,47 @@ The iOS jailbreak scene evolves so rapidly that providing up-to-date instruction
+#### Recommended Tools - iOS Device
 
-```http
-http://apt.thebigboss.org/repofiles/cydia/
-```
+Many tools on a jailbroken device can be installed by using Cydia, which is the unofficial AppStore for iOS devices and allows you to manage repositories. One of the most popular repositories is BigBoss, which contains various packages, such as the BigBoss Recommended Tools package. If your Cydia installation isn't pre-configured with this repository, you can add it by navigating to Sources -> Edit, then clicking "Add" in the top left and entering the following URL <http://apt.thebigboss.org/repofiles/cydia/>.
 
-You may also want to add the HackYouriPhone repository to get the AppSync package:
+You may also want to add the HackYouriPhone repository to get the AppSync package <http://repo.hackyouriphone.org>.
 
-```http
-http://repo.hackyouriphone.org
-```
+You can also easily install Frida by adding the following repository to Cydia <https://build.frida.re>.
 
 The following are some useful packages you can install from Cydia to get started:
 
 - BigBoss Recommended Tools: Installs many useful command line tools for security testing including standard Unix utilities that are missing from iOS, including wget, unrar, less, and sqlite3 client.
 - adv-cmds: Advanced command line. Includes finger, fingerd, last, lsvfs, md, and ps.
 - [IPA Installer Console](https://cydia.saurik.com/package/com.autopear.installipa/ "IPA Installer Console"): Tool for installing IPA application packages from the command line. Package name is `com.autopear.installipa`.
 - Class Dump: A command line tool for examining the Objective-C runtime information stored in Mach-O files.
-- Cydia or Mobile Substrate: A platform that makes developing third-party iOS add-ons easier via dynamic app manipulation or introspection.
-- cycript: Cycript is an inlining, optimizing, Cycript-to-JavaScript compiler and immediate-mode console environment that can be injected into running processes (assiciated to Substrate).
+- Mobile Substrate: A platform that makes developing third-party iOS add-ons easier via dynamic app manipulation or introspection.
+- Cycript: Is an inlining, optimizing, Cycript-to-JavaScript compiler and immediate-mode console environment that can be injected into running processes (associated to Substrate).
 - AppList: Allows developers to query the list of installed apps and provides a preference pane based on the list.
 - PreferenceLoader: A Mobile Substrate-based utility that allows developers to add entries to the Settings application, similar to the SettingsBundles that App Store apps use.
 - AppSync Unified: Allows you to sync and install unsigned iOS applications.
 
```

Comment:
Would it be a good idea to add apt.bigner.com as it will allow for adding apt install stuff through ssh?

Reply:
I added it, with the hint to only do it for Unv0ver jailbreaks. See also:
https://www.reddit.com/r/jailbreak/comments/9qfnmn/question_is_it_safe_to_do_updates_from/

Reply:
So the repo apt.binger.com is by default available:

```bash 
iPhone:/etc/apt root# cat /etc/apt/sources.list.d/cydia.list
deb https://apt.bingner.com/ ./
deb http://apt.thebigboss.org/repofiles/cydia/ stable main
deb http://cydia.zodttd.com/repo/cydia/ stable main
deb http://apt.modmyi.com/ stable main
deb https://repo.chariz.io/ ./
deb https://repo.dynastic.co/ ./
```